### PR TITLE
*: clean up the config template

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?

Clean up the `config.toml`

### What is changed and how it works?

Just as above says.

### Check List

Tests

- No code

### CODE REVIEW

**Review by JmPotato — COMMENTED** [review]
TSO part LGTM.

File: conf/config.toml [hunk 1]
```diff
@@ -88,84 +84,96 @@ cert-allowed-cn = ["example.com"]
...
-## The placement priorities is implied by the order of label keys.
+## The placement priorities are implied by the order of label keys.
 ## For example, ["zone", "rack"] means that we should place replicas to
 ## different zones first, then to different racks if we don't have enough zones.
-location-labels = []
+# location-labels = []
 ## Strictly checks if the label of TiKV is matched with location labels.
 # strictly-match-label = false
 
-[label-property]
```
Comment (JmPotato) [inline conf/config.toml:164]: Why delete it?

Comment (rleungx) [reply conf/config.toml:164]: Actually, we barely use it.

**Comment by rleungx** [comment]
@nolouch @HunDunDM PTAL

**Review by HunDunDM — APPROVED** [review]

**Review by JmPotato — APPROVED** [review]

**Pull request merged**